### PR TITLE
DDF-4164 Adding batchId to queries

### PR DESCRIPTION
**Pull request by Bdthomson**
#### What does this PR do?
- Adds batchId to queries generated from the frontend.
- Pull batchId off of request and place on request properties.
- Pull requestId off of request and place on request properties.

#### Who is reviewing it? 

@andrewzimmer
@Schachte 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@djblue 

#### How should this be tested?

View the outbound network traffic on the WS request and observe that each new request has a new batch id.

#### What are the relevant tickets?
[DDF-4164](https://codice.atlassian.net/browse/DDF-4164)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by Bdthomson** [comment]
build now (IO Error?)

**Comment by Bdthomson** [comment]
build now (dependency was deployed)

File: catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java [hunk 1]
```diff
@@ -172,6 +182,12 @@ public QueryRequest createQueryRequest(String localSource, FilterBuilder filterB
           .put(ADDITIONAL_SORT_BYS, sortBys.subList(1, sortBys.size()).toArray(new SortBy[0]));
     }
 
+    queryRequest.getProperties().put("requestId", id);
+
+    if (StringUtils.isNotEmpty(batchId)) {
+      queryRequest.getProperties().put("batchId", batchId);
```
Comment (stustison) [inline catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java]: Where is this value ever read from? Do we need any sort of validation around this field to ensure that it is unique and the correct format? What happens if these aren't unique? You can't rely on the UI to send an actual unique value to you without validating that it is unique on the back end.

**Comment by Bdthomson** [comment]
build now

**Comment by djblue** [comment]
Hero successful:
- Full build of `ui` and `catalog/ui/catalog-ui-search`
- Quick build
- Ensured batchId was in network requests

File: catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java (hunk 1 above)
Comment (pklinef) [reply catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java]: Correlation ID is the more common name for this.

**Pull request merged**